### ISSUE TITLE
86Box crashes under linux Mint

### ISSUE THREAD

**Issue by tthoms** [86Box/86Box#4779]
### What happened?

/86Box-Linux-x86_64-b6000.AppImage
Qt: version 5.15.2, platform "xcb"
libpng warning: IDAT: Extra compressed data
Fontconfig warning: "/usr/share/fontconfig/conf.avail/05-reset-dirs-sample.conf", line 6: unknown element "reset-dirs"
Segmentation fault (core dumped)

### Configuration file

```ini
Can't find 86box.cfg Maybe not generated outside of the AppImage?
```

### Operating system

Linux Mint 22 Cinnamon

### CPU

i5-8400H

### 86Box version

86-Box-Linux-x86_64-b6000

### Build architecture

Linux - x64 (64-bit)

### Build type

- [ ] New recompiler
- [ ] Debug build

### Download source

Official website (Jenkins, GitHub)

**Comment by OBattler** [comment]
Try the latest build on our Jenkins: https://ci.86box.net .

**Comment by eddmanx** [comment]
I don't know much about using linux, so don't take this as a definitive test however, I installed Mint 22 Cinnamon in VMware 16.2.5 and both builds 6000 and 6135 work OOTB.

**Comment by OBattler** [comment]
Yes, I think it's an user configuration problem - converting to a discussion.
